### ISSUE TITLE
Monitoring: engagement

### ISSUE THREAD

**Issue by eljhkrr** [medic/cht-app-monitoring-data-ingest#14]
% of users actively using the CHT platform represented by form submissions

**Comment by mrjones-plip** [comment]
I'm curious, when I look at the [monitoring API](https://docs.communityhealthtoolkit.org/apps/reference/api/#monitoring) there's both `couchdb.users.doc_count` and `connected_users.count` . Using those can you calculate engagement?

If not, can you quantify how you plan to calculate engagement  with forms submitted? Something like "if user submitted a report in past 7 days, they're active"?

**Comment by mrjones-plip** [comment]
>  how many docs are on the device which can be a good indicator for purging rules implementation [SNIP] Connected user count would need to be shown against the total count of expected active users but isn't actionable.

I think you're thinking of `couchdb.**medic**.doc_count` which would indeed be tied to docs on devices.  I'm referencing `couchdb.**users**.doc_count` which  will be a count of users on the system (but may include things like deleted users so might be off the older a deployment gets).  

There's clearly more than one way to do this, and I don't know which is best, but am trying to see if there's ways of doing things with the cheaper monitoring API calls vs more expensive authenticated calls.

**Comment by eljhkrr** [comment]
Right, I'm agreeing with you that connected users is indeed an engagement indicator but from a programs perspective we need to make this information actionable. The main question being "why are the CHWs not engaged". To investigate, there's a need to identify who they are and follow up on them directly which is why a table ranking has value. Connected users would be useful for measuring impact and there needs to be a mechanism to enable intervention where necessary.

**Comment by mrjones-plip** [comment]
> Right, I'm agreeing with you that connected users is indeed an engagement indicator but from a programs perspective we need to make this information actionable.

Gotcha - thanks!

**Comment by kennsippell** [comment]
@freddieptf This is the last P1 metric for App Monitoring. Any update on this? Would you like support on this task in April?

**Comment by eljhkrr** [comment]
This data point is returning 0's for almost all projects. Reopening to consider options to make it more usable.

**Comment by eljhkrr** [comment]
Gather related metrics and present as different widgets for interpretation by dashboard users
